lib/floe/workflow/intrinsic_function/transformer.rb

```diff
@@ -9,6 +9,11 @@ class Workflow
     class IntrinsicFunction
       class Transformer < Parslet::Transform
         OptionalArg = Struct.new(:type)
+        class Boolean
+          def self.to_s ; "Boolean" ; end
+          # NOTE: this is backwards
```

Comment:
Can you add more detail here?  What makes it backwards in your opinion?  Are there tests demostrating this backwardness?

Reply:
we do `Boolean.kind_of?(obj)` instead of `obj.kind_of?(Boolean)`

That is why we have a weird.

I have renamed to `cover?` and it reads a little better